server/graphql/v2/query/ExchangeRateQuery.js

```diff
@@ -0,0 +1,23 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { getExchangeRate } from '../../../lib/currency';
+import { Currency } from '../enum';
+
+const ExchangeRateQuery = {
+  type: GraphQLString,
```

Comment:
Is String a good choice here?

Reply:
Ops, I think `GraphQLFloat` is a better choice as what we are returning is actually a `float`. I've changed it to that. 👍🏽